util/HalideTraceViz.cpp

```diff
@@ -596,7 +613,7 @@ int run(int argc, char **argv) {
             }
 
             // Blank anim
-            memset(anim, 0, frame_bytes);
+            std::fill(anim.begin(), anim.end(), 0);
```

Comment:
Is std::fill known to be as fast as memset? I'm not sure what performance to expect from it. Trace visualization is slow, so it's worth checking the performance doesn't suffer more with this PR.